aea/configurations/base.py

```diff
@@ -301,14 +307,42 @@ def __init__(
         author: str,
         version: str,
         license: str,
+        aea_version: str = "",
         fingerprint: Optional[Dict[str, str]] = None,
     ):
-        """Initialize a package configuration."""
+        """
+        Initialize a package configuration.
+
+        :param name: the name of the package.
+        :param author: the author of the package.
+        :param version: the version of the package (SemVer format).
+        :param license: the license.
+        :param aea_version: either a fixed version, or a set of specifiers
+           describing the AEA versions allowed.
+           (default: empty string - no constraint).
+           The fixed version is interpreted with the specifier '=='.
+        :param fingerprint: the fingerprint.
+        """
         self.name = name
         self.author = author
         self.version = version
         self.license = license
         self.fingerprint = fingerprint if fingerprint is not None else {}
+        self.aea_version = aea_version if aea_version != "" else aea.__version__
+        self._aea_version_specifiers = self._parse_aea_version_specifier(aea_version)
+
+    def _parse_aea_version_specifier(self, aea_version_specifiers: str) -> SpecifierSet:
+        try:
+            Version(aea_version_specifiers)
```

Comment:
we first check the case `aea_version: <version_number>`.
It might fail since we allow for version specifiers.